### ISSUE TITLE
yad --mouse does not work on wayland

### ISSUE THREAD

**Issue by rickalex21** [v1cont/yad#261]
Hello,

Thanks for all the great work. I understand you don't use wayland as mentioned in #159 
but I wanted to bring up that `--mouse` does not work in case anyone else has
a wayland work around I would appreciate it. 

Thanks

**Comment by tmccombs** [comment]
I don't think that can generally be implemented in wayland, because a window can't control its own positioning.
